.github/workflows/test.yml

```diff
@@ -0,0 +1,86 @@
+name: Test Workflow
+
+on:
+  pull_request:
+    types: [opened, synchronize, reopened, edited, labeled, workflow_dispatch]
```

Comment:
`workflow_dispatch` should not under `types` of `pull_request`..

Reply:
Fixed, also removed labeled